pkg/openstack/loadbalancer.go

```diff
@@ -2542,7 +2532,7 @@ func (lbaas *LbaasV2) EnsureSecurityGroupDeleted(_ string, service *corev1.Servi
 	lbSecGroupName := getSecurityGroupName(service)
 	lbSecGroupID, err := secgroups.IDFromName(lbaas.network, lbSecGroupName)
 	if err != nil {
-		if isSecurityGroupNotFound(err) {
+		if cpoerrors.IsNotFound(err) {
```

Comment:
Random thought for @pierreprinetti @EmilienM @kayrus: I wonder if functions like this should be in gophercloud utils. We have basically the same in CAPO, which seems wasteful.

Reply:
Yes I think https://github.com/dulek/cloud-provider-openstack/blob/48a75a320f8ef9c9fe5d8c5bcd325ee47601ccb6/pkg/util/errors/errors.go#L45-L65 could definitely go in gophercloud/utils and would be welcomed.

Reply:
@EmilienM https://github.com/gophercloud/utils/pull/193